src/libtrx/game/objects/general/drawbridge.c

```diff
@@ -111,4 +127,87 @@ static void M_Setup(OBJECT *const obj)
     obj->floor_height_func = M_GetFloorHeight;
 }
 
+void M_DropStack(const ITEM *const item)
+{
+    VECTOR *pos = Vector_Create(sizeof(XYZ_32));
+    Drawbridge_GetSectorPositions(item, pos);
+    for (int32_t i = 0; i < pos->count; i++) {
+        MovableBlock_DropStack(
+            item->pos.y, *(const XYZ_32 *)Vector_Get(pos, i), item->room_num);
```

Comment:
Any chance of making this function more generic? Like, creating a proxy, `Item_DropStack` that calls `MovableBlock_DropStack`.
I'm thinking in the future it might be desirable for this logic to also affect pickup items, for example.

Reply:
I will think on how to best implement this for drop and slide. Unsure for now.